src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -699,6 +699,7 @@ private void CleanUpAllContainers()
 			{
 				CleanUpContainer(container);
 			}
+			ItemsPanelRoot?.Children?.Clear();
```

Comment:
Does this fix a separate bug for list view?

Reply:
**This is my behavior.**
1. Go to https://aka.platform.uno/wasm-samples-app
2. Search 'AutoSuggestBox' (search button is on the left bottom)
3. Select 'AutoSuggestBox_Keyboard'
4. Delete '0' of 'A0' in TextBox ( A0 -> A)
5. Add '0' ( A -> A0)
6. Press Down arror (  selecting should work )
7. Delete '0' and Add '0' again and Press Down arror ( selecting shouldn't work)

**Reason**
When ItemsSource are updated( i.e. containers are refreshed), some unnecessary containers are created.

the type of these unnecessary containers is object and they have not meaning.

Reply:
@Xiaoy312 , Could you review this change? Thank you.

Reply:
@TopProgrammer77  Thanks for the fix.

Should I be able to reproduce the issue here https://aka.platform.uno/wasm-samples-app? I am trying to reproduce it following the steps you provided but I am able to select.

I am using Firefox Developer Edition (114.0b3 (64-bit)) for my test

Reply:
Thanks @ajpinedam , If you can't reproduce on your side. you can check it on remote browser. https://live.browserstack.com/
I tested it on google 113.

Reply:
Hello, @ajpinedam . Did you reproduce the issue?

Reply:
@TopProgrammer77 Yes, I installed Chrome and was able to reproduce it.

Reply:
✅ @TopProgrammer77  Sure!